index.js

```diff
@@ -221,10 +221,12 @@ class DHT extends EventEmitter {
     this._backgroundQuery(node ? node.id : this.table.id).on('error', noop)
   }
 
-  destroy () {
+  async destroy () {
+    if (this.destroyed) return
```

Comment:
let the other destroys through still actually, the io.destroy streamlines them

but save a flag if this is the first one and the only emit close at the btm if its the first one